org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/problem/messages.properties

```diff
@@ -1154,7 +1154,7 @@
 # Record patterns = Java 21
 1912 = Only record types are permitted in a record pattern
 1913 = Record pattern should match the signature of the record declaration
-1914 = Pattern of type {0} is not compatible with type {1}
+1914 = Record component with type {0} is not compatible with type {1}
```

Comment:
An attempt at improving the message. @mpalat does this make sense? (see tests below for how it spells out in context).

Reply:
+1